src/code-components/ApiProvider/useOnError.ts

```diff
@@ -62,6 +62,17 @@ function handleApiError(
     }
   }
 
+  if (!error.handled && error.response?.status === 422 && toast) {
+    error.handled = true;
+    const message = await getMessage(error.response);
+    toast.show({
+      id: "unprocessable-content",
```

Comment:
This toast may actually be different per each request, with different error cause and message, so I'd rather keep them unique, by letting them be shown separately.

```suggestion
```